CollapseLauncher/XAMLs/MainApp/MainWindow.xaml

```diff
@@ -7,6 +7,9 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:customcontrol="using:CollapseLauncher.CustomControls"
     mc:Ignorable="d">
+    <Window.SystemBackdrop>
+        <DesktopAcrylicBackdrop/>
+    </Window.SystemBackdrop>
```

Comment:
@bagusnl Any reason why we added this acrylic effect to the main window? Since we are not using transparent/acrylic effect for the windows backdrop

Reply:
Oh, this was leftover from the attempt to fix the acrylic on flyout items, will remove